osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -436,7 +436,7 @@ protected override bool OnKeyDown(KeyDownEvent e)
                     break;
             }
 
-            if (direction == 0)
+            if (direction == 0 || e.AltPressed)
```

Comment:
It's unfortunate that this is how we have to do it, but I can't find a better way. Ultimately, we would want `OnkeyDown` and `OnPressed` to both be handled at the same time in a single tree iteration so we could correctly handle this at the receptor.

I think at very least, the early return should be moved to the top of this method, and commented with reasoning as to why it's done (so we can hopefully implement in a better way in the future).